client/app/bundles/course/assessment/submission/containers/TestCaseView.jsx

```diff
@@ -258,6 +273,7 @@ VisibleTestCaseView.propTypes = {
   showPrivate: PropTypes.bool,
   showEvaluation: PropTypes.bool,
   isAutograding: PropTypes.bool,
+  viewHistory: PropTypes.bool,
```

Comment:
The test case view should not be concerned with whether we are viewing history. What I can suggest instead is `collapsible` or something along that lines, so that when we deal with this component it is easier to read and understand.

Reply:
actually it does, `TestCaseView` is a container that gets its `testCases` from the store. If `viewHistory = true` then we will pull from the testCases in the history instead.